tools/tf2openapi/types/tf2openapi_tfsignaturedef_test.go

```diff
@@ -0,0 +1,95 @@
+package types
+
+import (
+	"github.com/kubeflow/kfserving/tools/tf2openapi/generated/framework"
+	pb "github.com/kubeflow/kfserving/tools/tf2openapi/generated/protobuf"
+	"github.com/onsi/gomega"
+	"testing"
+)
+
+/* Expected values */
+func expectedTFSignatureDef() TFSignatureDef {
+	return TFSignatureDef{
+		Key: "Signature Def Key",
+		Inputs: []TFTensor{
+			{
+				Name:  "input",
+				DType: DtInt8,
+				Shape: TFShape{-1, 3},
+				Rank:  2,
+			},
+		},
+		Outputs: []TFTensor{
+			{
+				Name:  "output",
+				DType: DtInt8,
+				Shape: TFShape{-1, 3},
+				Rank:  2,
+			},
+		},
+	}
+}
+
+/* Fake protobuf structs to use as test inputs */
+func tensorsPb(name string, willErr bool) map[string]*pb.TensorInfo {
```

Comment:
I'd suggest split this methods into a `goodTensorsPb(...)` method, and a `badTensorsPb(...)` method. Again, the actual naming is up to you.

It's normally discouraged to pass a boolean to a function for a broad if/else logic. The method has two responsibility now, which could be easily expressed with two functions with better readability, especially at the caller level. (e.g. in this file, the test cases will be easier to read, since readers don't need to guess or look up for what `true` or `false` means.)

Reply:
Got it!